test/request_source/test_data/test-config-ab.yaml

```diff
@@ -1,11 +1,13 @@
 options:
   - request_method: 1
     request_body_size: 10
+    json_body: '{"message": "hello"}'
```

Comment:
Could we set different body values in the two requests and check both of them in the test, along with checking /a and /b?